dist/linux/appimage.sh

```diff
@@ -6,8 +6,8 @@ fi
 
 curl -sSfLO "https://github.com/linuxdeploy/linuxdeploy/releases/download/continuous/linuxdeploy-x86_64.AppImage"
 chmod a+x linuxdeploy*.AppImage
-curl -sSfL https://github.com"$(curl https://github.com/probonopd/go-appimage/releases/expanded_assets/continuous | grep "mkappimage-.*-x86_64.AppImage" | head -n 1 | cut -d '"' -f 2)" -o mkappimage.AppImage
-chmod a+x mkappimage.AppImage
+curl -sSfL "https://github.com/AppImage/appimagetool/releases/download/continuous/appimagetool-x86_64.AppImage" -o appimagetool
+chmod a+x appimagetool
```

Comment:
The reason we use `mkappimage` instead of `appimagetool` is because of the change of libfuse2 no longer standard on many newer distros. mkappimage statically links this, avoiding another requirement for the host system.

Reply:
Good that's what I meant with the static runtime. AppImage/appimagetool also pulls the static runtime now.